### PR TITLE
530: Bug - Update The Revisions Widget

### PR DESCRIPTION
**Pull request by ajvillegas**
## Issue Number
Issue humanmade/hm-playbook-theme#530

## Description
This change updates the Revisions Widget style bugs and date format.

| Default Style | Hover & Focus |  Selected Revision |
| ----------- | ----------- | ----------- |
| Screen Shot 2022-12-23 at 2 28 04 PM     | Screen Shot 2022-12-23 at 2 28 18 PM      |Screen Shot 2022-12-23 at 2 28 39 PM     |

## Steps to Test
1. Navigate to a page with a Revision widget.
2. Test the "Load Revision" button and make sure it displays with a border and color background on focus and hover.
3. Check that the date is displayed in YYYY-MM-DD format.

## Acceptance Criteria

- [x] The widget uses YYYY-MM-DD to align with dates formats in the rest of the Playbook
- [x] ~The widget does not include the time of the revision~ Clean up the styling for this to make it more readable 
- [x] There is appropriate spacing between the last revision event and the `Load Revisions` CTA within the widget.
- [x] Has this been tested in multiple browsers
- [x] Has it been accessibility tested
- [x] Have all linters and PHPCS checks been run
- [x] Does the README.md and the Wiki need updating to reflect these changes
- [x] Do any articles on how to update the Playbook in https://playbook.hmn.md/category/updating-the-playbook/ need updating to reflect these changes.

### CODE REVIEW

File: assets/src/styles/index.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-.hm-post-history {
-	--post-history-red: var( --post-history-red, #ff424a );
+.post-history {
+	--post-history-red: #ff424a;
```
Comment (kadamwhite) [inline assets/src/styles/index.scss]: The purpose of this rule was to allow this to be overridden by a consuming project -- why was the fallback removed?

**Review by kadamwhite — APPROVED** [review]
Most changes look good except I'd prefer we keep the ability to default to a custom property defined further up, which is what I believe some of the removed code does. Clear to merge to dev though

**Comment by kadamwhite** [comment]
As an aside; when referencing tickets in another repo, its more useful to use the format `org/repo#issue` than `#issue` alone. That way GH will cross link them and also you won't get confused in another few hundred tickets when 530 actually means something in this repository. Example: humanmade/hm-playbook-theme#530

(Edited to fix the link, which I typo'd the first time!)

File: assets/src/styles/index.scss (hunk 1 above)
Comment (ajvillegas) [reply assets/src/styles/index.scss]: The CSS var was not getting defined in the browser and not rendering. It seems that you can't name a var the same as the value like `--post-history-red: var( --post-history-red, #ff424a )`.

I instead renamed it something different to allow for `--post-history-red` to be defined somewhere else and thus overwritten:

```
--post-history-accent: var( --post-history-red, #ff424a );
```

**Pull request merged**